docs/download.md

```diff
@@ -85,24 +85,27 @@ CUDA 11.8 through CUDA 12.0.
 * Download the [PUB_KEY](https://keys.openpgp.org/search?q=sw-spark@nvidia.com).
 * Import the public key: `gpg --import PUB_KEY`
 * Verify the signature for Scala 2.12 jar:
-    `gpg --verify rapids-4-spark_2.12-24.10.1.jar.asc rapids-4-spark_2.12-24.10.1.jar`
+    `gpg --verify rapids-4-spark_2.12-24.12.0.jar.asc rapids-4-spark_2.12-24.12.0.jar`
 * Verify the signature for Scala 2.13 jar:
-    `gpg --verify rapids-4-spark_2.13-24.10.1.jar.asc rapids-4-spark_2.13-24.10.1.jar`
+    `gpg --verify rapids-4-spark_2.13-24.12.0.jar.asc rapids-4-spark_2.13-24.12.0.jar`
 
 The output of signature verify:
 
 	gpg: Good signature from "NVIDIA Spark (For the signature of spark-rapids release jars) <sw-spark@nvidia.com>"
 
 ### Release Notes
-* Optimize scheduling policy for GPU Semaphore
-* Support distinct join for right outer joins
-* Support MinBy and MaxBy for non-float ordering  
-* Support ArrayJoin expression
-* Optimize Expand and Aggregate expression performance
-* Improve JSON related expressions
+* Implement Kudo shuffle format to improve shuffle performance
+* Add the ability to decompress snappy and ztsd compressed Parquet files via CPU
+* Add repartition-based algorithm fallback in hash aggregate  
+* Support spark function months_between
+* Support asynchronous writing for Parquet files
+* Add retry support to improve sub hash-join stability
+* Improve JSON scan and from_json
+* Support setting a limited number of tasks for build-in profiling tool
+* Bugs fix for memory leak
```

Comment:
Ideally this should be a bit more descriptive.  Is it a host or device memory leak?  What are the conditions for the leak, i.e.: what types of operations were affected?

Reply:
Double check the PRs that we don't have a memory leak bug fix this release, will remove it.

Reply:
We have at least this one: https://github.com/NVIDIA/spark-rapids/pull/11848/commits/3449c8a772899ff1752ea42efa89f1ddc049cc6a

Reply:
and this one https://github.com/NVIDIA/spark-rapids/pull/11848/commits/96a58d121a5af7ef956196a6141fe7777277e95e

Reply:
updated, thx